lib/pf/dal/api_users.pm

```diff
@@ -0,0 +1,49 @@
+package pf::dal::api_users;
```

Comment:
Same thing/question - api_users.pm or api_user.pm?

Reply:
Unless we are planning to move to a new naming convention we should stay consistent.

Reply:
True, let me call my friend sed to get that fixed